service/create/service.go

```diff
@@ -462,7 +463,7 @@ func validateIDs(ids []string) bool {
 
 func (s *Service) addFunc(obj interface{}) {
 	cluster := *obj.(*awstpr.CustomObject)
-	s.logger.Log("info", fmt.Sprintf("creating cluster '%s'", cluster.Name))
+	s.logger.Log("info", fmt.Sprintf("creating cluster '%s'", key.ClusterID(cluster)))
```

Comment:
Yes having the logging including the cluster ID without a lot of duplicate code would be very useful.